samples/bluetooth/hci_rpmsg/nrf5340_cpunet_iso_receive.conf

```diff
@@ -22,22 +22,24 @@ CONFIG_BT_OBSERVER=y
 CONFIG_BT_CENTRAL=n
 CONFIG_BT_EXT_ADV=y
 # CONFIG_BT_PER_ADV=y
-# CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_PER_ADV_SYNC=y
+# CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_MAX_CHAN=2
 
 # ISO Broadcast Controller
-# CONFIG_BT_CTLR_ADV_EXT=y
 # CONFIG_BT_CTLR_ADV_PERIODIC=y
 # CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
 # CONFIG_BT_CTLR_ADV_ISO=y
 # CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=251
-# CONFIG_BT_CTLR_ISO_TX_BUFFERS=3
+# CONFIG_BT_CTLR_ISO_TX_BUFFERS=16
```

Comment:
Shouldn't all these related to TX be removed for this config?

Reply:
These are here for reference and to be able to enable combined broadcast+receive configuration without doing a new `cmake` call.

Reply:
I am not sure I understand. For an `iso_receive` config, it doesn't really make much sense to keep the ISO broadcaster configs here as a reference, especially since we also have the `iso_broadcast` config file right next to it. 

If we want to have a config for a combined build, why not just create that config file instead?